### PR TITLE
fixedFullFilter

### PR DESCRIPTION
**Pull request by RomchikSt**
Trello: https://trello.com/c/MQQBa3Nl/77-fixed-type-venchicle-and-studded-filters

FilterShortMenuContainer - 128 - type any
IDK wich type must be there(

After:

Uploading Screen Recording 2024-02-06 at 17.29.53.mp4…

### CODE REVIEW

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const StyledButtonMain = styled(Button)({
   '&:active': {
     backgroundColor: 'transparent',
     borderColor: FILTER_COLORS.BORDER,
-    borderTop: 'none',
+    borderBoborderTopttom: 'none',
```
Comment (unreal998) [inline src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: wtf `borderBoborderTopttom ` ?)

File: src/modules/shopPage/components/ShopContainer.tsx [hunk 2]
```diff
@@ -121,7 +121,6 @@ export function ShopContainer() {
         </Box>
       ) : (
         <>
-          {/* <NoFilterResults /> */}
```
Comment (unreal998) [inline src/modules/shopPage/components/ShopContainer.tsx:124]: remove commented code

**Review by unreal998 — CHANGES_REQUESTED** [review]

File: src/modules/shopPage/components/ShopContainer.tsx (hunk 2 above)
Comment (RomchikSt) [reply src/modules/shopPage/components/ShopContainer.tsx:124]: he was removed

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx (hunk 1 above)
Comment (RomchikSt) [reply src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: done

**Review by unreal998 — APPROVED** [review]
LGTM

**Pull request merged**